components/sponsors/SponsorModal.vue

```diff
@@ -96,8 +98,12 @@ export default {
     @apply flex items-center;
 }
 
+.lightBox__subtitle {
+    color: #f3cc39;
+}
+
 .lightBox__intro {
-    @apply flex mb-4 overflow-y-scroll;
+    @apply flex mt-2 mb-4 overflow-y-scroll;
```

Comment:
```suggestion
    @apply flex mt-3.5 mb-4 overflow-y-scroll;
```

<img width="762" alt="figma" src="https://user-images.githubusercontent.com/7352613/124386744-65ac3f80-dd0e-11eb-80a1-c50dd8c8ce69.png">